pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[build-system]
+requires = ["hatchling>=1.25"]
+build-backend = "hatchling.build"
+
+[project]
+name = "model_signing"
```

Comment:
the pypi link redirects to "model-signing". Should we make that change?

Reply:
It shouldn't matter, https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#name says that `_`, `-`, and `.` are equivalent

Reply:
This does affect how the project name will be displayed at https://pypi.org/project/model-signing/, however.

Reply:
Oh, then I'll change to the dash. Thank you!